src/server/index.js

```diff
@@ -19,6 +19,7 @@ const express = require('express')
 const compression = require('compression')
 const { s3Url } = require('./config')
 const { isProduction } = require('./utils')
+require('dotenv').config()
```

Comment:
? do we need this?

Reply:
We need it to use `dotenv` with the server, which helps a lot in setting up the required environment variables like the GitHub token in a dev environment without having to manually re-add the API key to the command invocation each time.

As far as if it's necessary to the functionality, it isn't, it's just a harmless but big improvement to dev QOL.

Reply:
yeah, just curious why specifically this place

Reply:
Totally! It's weird to see it in three places including here, but there's a reason.

Reply:
👍